### ISSUE TITLE
Build warnings

### ISSUE THREAD

**Issue by nohli** [RevenueCat/purchases-flutter#95]
**Describe the bug**
Build warnings

**Environment**
    1. Platform: iOS
    2. SDK version: 2.9.0
    3. OS version: 13.7
    4. XCode/Android Studio version: 11

**Additional context**
```
[00:14:02]: ▸ ⚠️  /Users/.../code/.../ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: method definition for '_setPushTokenString:' not found [-Wincomplete-implementation]
[00:14:02]: ▸ @implementation RCPurchases (HybridAdditions)
[00:14:02]: ▸                 ^
[00:14:02]: ▸ ⚠️  /Users/.../code/.../ios/Pods/PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: method definition for 'configureWithAPIKey:appUserID:observerMode:userDefaults:platformFlavor:platformFlavorVersion:' not found [-Wincomplete-implementation]
[00:14:02]: ▸ @implementation RCPurchases (HybridAdditions)
[00:14:02]: ▸ ^
```

**Comment by aboedo** [comment]
hi! 👋 thanks for opening. Those warnings should be safe to ignore for now, but we're going to do some refactors so that they no longer show. Sorry for the annoyance.

**Comment by erf** [comment]
I also get the following warning when building for iOS:

```
/Users/erlend/.pub-cache/hosted/pub.dartlang.org/purchases_flutter-1.4.3/ios/Classes/PurchasesFlutterPlugin.m:197:28: 'addAttributionData:network:networkUserId:' is deprecated: Use the set<NetworkId> functions instead.
```

**Comment by aboedo** [comment]
@nohli @erf this has been fixed in version `2.0.0`, which has just been released. Thanks again for reporting! 🎉
